<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/img/backgroundFaded_bee.png);</string>
    </property>
    <widget class="QWidget" name="HomePage">
     <widget class="QPushButton" name="LevelSelectButton">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>170</y>
        <width>262</width>
        <height>112</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/Button.png);
font: 28pt Candyshop;
border: none;</string>
      </property>
      <property name="text">
       <string>Level Select</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CheckCameraButton">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>340</y>
        <width>262</width>
        <height>112</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/Button.png);
font: 28pt Candyshop;
border: none;</string>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SettingsButton">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>510</y>
        <width>261</width>
        <height>112</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/Button.png)  0 0 0 0 stretch stretch;
font: 26pt Candyshop;
border: none;</string>
      </property>
      <property name="text">
       <string>Change Theme</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="LevelSelect">
     <widget class="QPushButton" name="Level1">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>180</y>
        <width>110</width>
        <height>112</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/SmallButton.png);
border:none;
font: 64pt &quot;Candyshop&quot;;</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Level2">
      <property name="geometry">
       <rect>
        <x>508</x>
        <y>180</y>
        <width>110</width>
        <height>112</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/SmallButton.png);
border:none;
font: 64pt &quot;Candyshop&quot;;</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Level3">
      <property name="geometry">
       <rect>
        <x>676</x>
        <y>180</y>
        <width>110</width>
        <height>112</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/SmallButton.png);
border:none;
font: 64pt &quot;Candyshop&quot;;</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Level4">
      <property name="geometry">
       <rect>
        <x>844</x>
        <y>180</y>
        <width>110</width>
        <height>112</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/SmallButton.png);
border:none;
font: 64pt &quot;Candyshop&quot;;</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Level5">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>350</y>
        <width>110</width>
        <height>112</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/SmallButton.png);
border:none;
font: 64pt &quot;Candyshop&quot;;</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Level6">
      <property name="geometry">
       <rect>
        <x>508</x>
        <y>350</y>
        <width>110</width>
        <height>112</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/SmallButton.png);
border:none;
font: 64pt &quot;Candyshop&quot;;</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Level7">
      <property name="geometry">
       <rect>
        <x>676</x>
        <y>350</y>
        <width>110</width>
        <height>112</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/SmallButton.png);
border:none;
font: 64pt &quot;Candyshop&quot;;</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Level8">
      <property name="geometry">
       <rect>
        <x>844</x>
        <y>350</y>
        <width>110</width>
        <height>112</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/SmallButton.png);
border:none;
font: 64pt &quot;Candyshop&quot;;</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Level9">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>520</y>
        <width>110</width>
        <height>112</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/SmallButton.png);
border:none;
font: 64pt &quot;Candyshop&quot;;</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Level10">
      <property name="geometry">
       <rect>
        <x>508</x>
        <y>520</y>
        <width>110</width>
        <height>112</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/SmallButton.png);
border:none;
font: 64pt &quot;Candyshop&quot;;</string>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Level11">
      <property name="geometry">
       <rect>
        <x>676</x>
        <y>520</y>
        <width>110</width>
        <height>112</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/SmallButton.png);
border:none;
font: 64pt &quot;Candyshop&quot;;</string>
      </property>
      <property name="text">
       <string>11</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Level12">
      <property name="geometry">
       <rect>
        <x>844</x>
        <y>520</y>
        <width>110</width>
        <height>112</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/SmallButton.png);
border:none;
font: 64pt &quot;Candyshop&quot;;</string>
      </property>
      <property name="text">
       <string>12</string>
      </property>
     </widget>
     <widget class="QPushButton" name="HomeButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>110</width>
        <height>112</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/Home.png);
border:none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="CameraPage">
     <widget class="QPushButton" name="HomeButton2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>110</width>
        <height>112</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/Home.png);
border:none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>140</y>
        <width>640</width>
        <height>480</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">/*background-image: url(:/img/deleteMe.png);*/
font: 72pt &quot;Candyshop&quot;;
color: rgb(255, 0, 255);</string>
      </property>
      <property name="text">
       <string>Camera Feed (this text should not appear)</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="SettingsPage">
     <widget class="QPushButton" name="HomeButton3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>110</width>
        <height>112</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/Home.png);
border:none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="setThemeKnight">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>410</y>
        <width>261</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/Button.png);
font: 28pt Candyshop;
border: none;</string>
      </property>
      <property name="text">
       <string>Knight</string>
      </property>
     </widget>
     <widget class="QPushButton" name="setThemeBee">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>280</y>
        <width>261</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/Button.png);
font: 28pt Candyshop;
border: none;</string>
      </property>
      <property name="text">
       <string>Bee</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>90</y>
        <width>261</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/Button.png);
font: 28pt Candyshop;
border: none;</string>
      </property>
      <property name="text">
       <string>Select Theme</string>
      </property>
     </widget>
     <widget class="QPushButton" name="setThemeKnight_2">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>540</y>
        <width>261</width>
        <height>111</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/Button.png);
font: 28pt Candyshop;
border: none;</string>
      </property>
      <property name="text">
       <string>Space</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Game">
     <widget class="QPushButton" name="HomeButton4">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>110</width>
        <height>112</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/Home.png);
border:none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QGraphicsView" name="GameView">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1280</width>
        <height>720</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
     </widget>
     <widget class="QLabel" name="LevelCompleteBox">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>60</y>
        <width>621</width>
        <height>601</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/HowToPlayBackground.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="WellDone">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>140</y>
        <width>551</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:none;
font: 48pt &quot;Candyshop&quot;;</string>
      </property>
      <property name="text">
       <string>Level Finished</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="NextLevelQuestion">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>385</y>
        <width>601</width>
        <height>90</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:none;
font: 22pt &quot;Candyshop&quot;;</string>
      </property>
      <property name="text">
       <string> Try Again                   Next Level</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="NextLevel">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>460</y>
        <width>110</width>
        <height>112</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/SmallButton.png);
font: 48pt &quot;Candyshop&quot;;
border: none;</string>
      </property>
      <property name="text">
       <string>✔</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ExitLevel">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>460</y>
        <width>110</width>
        <height>112</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/SmallButton.png);
font: 48pt &quot;Candyshop&quot;;
font-weight: 1000;
border: none;</string>
      </property>
      <property name="text">
       <string>↺</string>
      </property>
     </widget>
     <widget class="QLabel" name="LevelLabel">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>20</y>
        <width>300</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:none;
font: 48pt &quot;Candyshop&quot;;</string>
      </property>
      <property name="text">
       <string>Level 1</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="ScoreLabel">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>270</y>
        <width>581</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: none;
font: 26pt &quot;Candyshop&quot;;
color: black;
</string>
      </property>
      <property name="text">
       <string>You took %1 Moves</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="BestScoreLabel">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>330</y>
        <width>360</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background:gold;
font: 26pt &quot;Candyshop&quot;;
color: black;
border-radius:15px;</string>
      </property>
      <property name="text">
       <string>Best score %2 Moves</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>510</y>
        <width>846</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/img/warningBox.png);
font: 28pt &quot;Candyshop&quot;;</string>
      </property>
      <property name="text">
       <string>
No Grid Found.
Do you have your hand in the way?</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
     <zorder>label_2</zorder>
     <zorder>GameView</zorder>
     <zorder>HomeButton4</zorder>
     <zorder>LevelCompleteBox</zorder>
     <zorder>WellDone</zorder>
     <zorder>NextLevelQuestion</zorder>
     <zorder>NextLevel</zorder>
     <zorder>ExitLevel</zorder>
     <zorder>LevelLabel</zorder>
     <zorder>ScoreLabel</zorder>
     <zorder>BestScoreLabel</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
